File: Ulbra 2023-1/Algortimos e Programação/Aula 06 (Estruturas de decisão mais dificeis)/exerc5.py
#5. Faça um programa que leia o código de um aluno e suas três notas. Calcule a média ponderada do aluno, considerando que o peso para a maior nota seja 4 e para as duas restantes, 3. Mostre o código do aluno, suas três notas, a média calculada e a mensagem “ APROVADO” se a média for maior ou igual a 7 e “REPROVADO” se a média for menor que 7.
aluno = str(input("Digite o código do aluno: "))
nota1 = float(input("Digite a nota da primeira prova: "))
nota2 = float(input("Digite a nota da segunda prova: "))
nota3 = float(input("Digite a nota da terceira prova: "))
media = ((nota1*4) + (nota2*3) + (nota3*3)) / 10
media = round(media, 2)
if media > 10 or media < 0:
    print("Nota inválida!")
elif media >= 7:
    print(f"O aluno de código {aluno}, está APROVADO com média de {media} em suas provas ({nota1}, {nota2} e {nota3}).")
elif media < 7:
    print(f"O aluno de código {aluno}, está REPROVADO com média de {media} em suas provas ({nota1}, {nota2} e {nota3}).")

File: Ulbra 2023-1/Introdução ao Desenvolvimento Web/Aula 17 (JavaScript)/exerc5.js
//5 - Crie um script que leia três notas e imprima a média das notas e informe se o aluno está aprovado ou não. A média para aprovação é 6.

let nota1 = Number(prompt("Digite a primeira nota: "))
let nota2 = Number(prompt("Digite a segunda nota: "))
let nota3 = Number(prompt("Digite a terceira nota: "))
media = (nota1 + nota2 + nota3) / 3

if (media >= 6.0){
    alert(`Aluno aprovado com média: ${media}.`)
}else{
    alert(`Aluno reprovado com média: ${media}.`)
}